src/openzaak/urls.py

```diff
@@ -45,3 +46,7 @@
     import debug_toolbar
 
     urlpatterns += [path("__debug__/", include(debug_toolbar.urls))]
+
+
+if apps.is_installed("silk"):
```

Comment:
Oh, nice. Should we also use this pattern for debug toolbar?

Reply:
yes :-)